python/contactflowhelper/lambda_function.py

```diff
@@ -0,0 +1,114 @@
+'''
+Basic helper function that covers common text/math operations for use with 
+contact flows. This function is assuming incoming events from a contact flow. 
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import json
+import random
+
+def lambda_handler(event, context):
+    response = {}
+    
+    if event['Details']['Parameters']['function'] == 'freeform_math':
+        response.update({'answer':str(freeform_math(event))})
```

Comment:
another thing to clean up - create a variable "parameters = event['Details']['Parameters']" and pass that into each of the functions. That makes the functions shorter / easier to read as you will just do "parameters['expression']"

Reply:
fixed